bin/otomi

```diff
@@ -101,21 +99,16 @@ function show_usage() {
 }
 
 function check_docker_running() {
-  set +e
-  docker --version &>/dev/null
-  set -e
-  if [ "$?" = "0" ]; then
-    has_docker=1
-  else
-    echo "Otomi CLI expects a running docker daemon!" >&2 && exit 1
+  if [ "${has_docker}" = "0" ]; then
```

Comment:
Please remove this pointless function, and print the message where this is invoked

Reply:
https://github.com/redkubes/otomi-core/blob/master/bin/otomi#L103
https://github.com/redkubes/otomi-core/blob/master/bin/otomi#L403

Again, keeping codestyle close to original. Will change